raw128.go

```diff
@@ -267,7 +267,7 @@ func ult128(a, b raw128) bool {
 func slt128(a, b raw128) bool {
 	if isEqual64(a.Hi, b.Hi) {
 		// If the high parts are equal, compare the low parts.
-		return slt64(a.Lo, b.Lo)
+		return ult64(a.Lo, b.Lo)
```

Comment:
oh so the bug was that `slt64()` converts the `uint64` low-bits to `int64` and thats a lossy-conversion?